gnovm/pkg/gnolang/preprocess.go

```diff
@@ -9,7 +9,10 @@ import (
 	"github.com/gnolang/gno/tm2/pkg/errors"
 )
 
-const blankIdentifer string = "_"
+const (
+	blankIdentifier    string = "_"
+	blankNameIdentifer Name   = "_"
```

Comment:
Why do we need 2 of these?

Reply:
There are two types. I think it is nicer than casting each `Name` typed blank identifier literal.

Reply:
What if we just kept `blankIdentifier` as an untyped const (ie. with neither `string` nor `Name`) so it just gets the type inferred (without having to do conversions)?

Reply:
done